### PR TITLE
Ordinance lab pumps possibly cosmetic change.

### PR DESCRIPTION
**Pull request by orthography**
## About The Pull Request

Adds the possibility to change the pumps in the ordinance labs of any map.

## Why It's Good For The Game

Silly addition to the ordinance lab with no real gameplay change, purely cosmetic.

Maybe people will stop hating me for making it only 1 "Lil' Pump" per map 

## Changelog

:cl:
add: Adds a new possible cosmetic change to the ordinance lab.
/:cl:

### CODE REVIEW

File: _maps/map_files/generic/CentCom.dmm [hunk 1]
```diff
@@ -2307,10 +2307,7 @@
 /area/centcom/central_command_areas/briefing)
 "gG" = (
 /obj/effect/turf_decal/stripes/box,
-/obj/machinery/portable_atmospherics/pump{
-	desc = "A betrayer to pump-kind.";
-	name = "Lil Pump"
-	},
+/obj/machinery/portable_atmospherics/pump/lil_pump,
```
Comment (san7890) [inline _maps/map_files/generic/CentCom.dmm:2310]: Would you mind going back and adding the description to the one on the syndicate lavaland base?

**Review by san7890 — CHANGES_REQUESTED** [review]

**Comment by Striders13** [comment]
awesome

File: _maps/map_files/generic/CentCom.dmm [hunk 2]
```diff
@@ -2307,9 +2307,9 @@
 /area/centcom/central_command_areas/briefing)
 "gG" = (
 /obj/effect/turf_decal/stripes/box,
-/obj/machinery/portable_atmospherics/pump{
+/obj/machinery/portable_atmospherics/pump/lil_pump{
 	desc = "A betrayer to pump-kind.";
```
Comment (san7890) [inline _maps/map_files/generic/CentCom.dmm:2311]:
```suggestion
/obj/machinery/portable_atmospherics/pump/lil_pump{
	desc = "A betrayer to pump-kind."
}
```
The map key appears to have screwed up during your edit, just remove the semicolon and ensure that there are no newlines within the variables associated to an object.

Comment (orthography) [reply _maps/map_files/generic/CentCom.dmm:2311]: Copilot has fucked me.

**Pull request merged**